modules/persistent-storage-csi-gcp-filestore-nfs-export-options.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-filestore-nfs-export-options_{context}"]
+= NFS export options
+
+Network File System (NFS) sets file share permissions when creating a storage class. Using NFS export options propagate these options to the server for the Filestore instance.
+
+.Prerequisites
+* Access to the cluster as a user with the cluster-admin role.
+
+* The {gcp-short} Filestore CSI Driver Operator and {gcp-short} Filestore CSI driver installed.
+
+.Procedure
+
+. Create a storage class using a file similar to the following sample YAML file:
++
+[NOTE]
+====
+For more information about creating a storage class, see Section _Creating a storage class for GCP Filestore Operator_.
+====
++
+.Example storage class YAML file with NFS export options
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+ name: SC-name
+provisioner: SC-provisioner
+parameters:
+ connect-mode: DIRECT_PEERING
+ network: project-network
+ nfs-export-options-on-create: '[ <1>
+   {
+     "accessMode": "READ_WRITE", <2>
+     "squashMode": "NO_ROOT_SQUASH", <3>
+     "ipRanges": [ <4>
+       "10.0.0.0/16"
+     ]
+   }]'
+allowVolumeExpansion: true
+----
+<1> *NFS export options parameter*: `nfs-export-options-on-create` has to start with '[ and end with }]', otherwise YAML cannot parse it. Also, parameters must be in ​​quotations marks, otherwise a parsing error occurs during persistent volume claim (PVC) creation.
```

Comment:
```suggestion
<1> *NFS export options parameter*: `nfs-export-options-on-create` must start with `'[ and end with }]'`, otherwise YAML cannot parse it. Also, parameters must be in ​​quotations marks, otherwise a parsing error occurs during persistent volume claim (PVC) creation.
```
Should '[ and end with }]' be in back ticks or ' ' should be replaced with back ticks (`[ and end with }]`)  ?